contracts/VetoMultisigVoting.sol

```diff
@@ -0,0 +1,243 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+import "./interfaces/IVetoGuard.sol";
+import "./interfaces/IVetoVoting.sol";
+import "./interfaces/IGnosisSafe.sol";
+import "./TransactionHasher.sol";
+import "@gnosis.pm/safe-contracts/contracts/common/Enum.sol";
+import "@gnosis.pm/zodiac/contracts/factory/FactoryFriendly.sol";
+
+/// @notice A contract for casting veto votes with an ERC20 votes token
+contract VetoMultisigVoting is IVetoVoting, TransactionHasher, FactoryFriendly {
```

Comment:
I did consider this, since both the VetoMultisigVoting and VetoERC20Voting contracts implement freeze functionality, and at first glance both could inherit from a common abstract freeze contract.

However, I didn't go down this path, since there are differences in the way those two contracts implement the freeze functionality. For example, the way the freeze votes are counted differs between the two (token votes at a historical block vs just being a multisig signer represents having one vote)